cirq/google/engine/engine.py

```diff
@@ -150,6 +153,7 @@ def __init__(
                                     service_args=service_args,
                                     verbose=verbose)
         self.context = context
```

Comment:
does it make sense to put this on the context?

Reply:
Sounds like a good idea to me.  Done.